### PR TITLE
Brought in landscape-tile doc post docs repo reorg

### PR DESCRIPTION
**Pull request by baudtack**

### CODE REVIEW

File: tutorials/hoon/landscape-tile.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+
+## Our Own Tile
+
+Let's examine a sample tile that has different Hoon code from the default, to explore how the tile system works. This program, composed of the `testing.hoon` and `tile.js` files, creates a tile which can be used to send `hi` messages to ships, just the way you can with `|hi ~zod` from the dojo. Keep in mind that unless you boot other fakeships you won't get responses from ships other than `~zod`.
+
+`tile.js` does the actual front-end rendering using React. The details of React are left for the reader to learn on their own, but you can see here how to store data in the state. That tile will be rendered every time the state gets updated. Calling `api.action` will send a JSON poke to our gall app. We have attached the function where we use that to the button on the tile, so it will get run when the button is clicked.
+
+In your pier, replace the code in `/app/testing.hoon` with the Hoon code below.
+
+```
```
Comment (matildepark) [inline tutorials/hoon/landscape-tile.md]: add 'hoon'

File: tutorials/hoon/landscape-tile.md [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+  ?:  =(name 'tile')
+    [[ost.bol %http-response (js-response:app tile-js)]~ +>.$]
+  [[ost.bol %http-response not-found:app]~ +>.$]
+::
+--
+```
+
+Now, replace the code below is the contents of `/app/testing/js/tile.js` with the code below. Once you've replaced the code in both files, run `|commit %home` in the Dojo.
+
+```
```
Comment (matildepark) [inline tutorials/hoon/landscape-tile.md]: add 'javascript'

File: tutorials/hoon/landscape-tile.md [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+
+### Default Code Components
+
+Parts of the `testing.hoon` code shown above is the "scaffolding" that was automatically built for us by the default Hoon file from the `create-landscape-app` repo. We'll give a general overview of those arms here.
+
+As with other gall apps, `card` and `move` define the types that can be produced by the app, `card` specifically defines which type of requests can be made to other parts of the system.
+
+`++prep` is called when the application first starts, and when its code gets updated. It will get a `unit` of the old state and `++prep` needs to update the state. You might notice the use of `~&`. This is useful to give us feedback to make sure our changes have been loaded.
+
+The `++peer-testingtile` arm is called when the tile first subscribes to the app. The subscription logic is mostly handled for us by the `launch` app here it simply produces some data to send back to the tile, indicating its initial state.
```
Comment (ohAitch) [inline tutorials/hoon/landscape-tile.md]: What specifies the subscription being opened, is it keyed off the `testingTile` class name? Also, does "initial `state`" here mean `props.data`, in react terms?

File: tutorials/hoon/landscape-tile.md [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+  [~ +>.$]
+++  prep
+  |=  old=(unit ~)
+  ~&  'it built'
+  ^-  (quip move _+>.$)
+  =/  launcha
+    [%launch-action [%testing /testingtile '/~testing/js/tile.js']]
+  :_  +>.$
+  :~
+    [ost.bol %connect / [~ /'~testing'] %testing]
```
Comment (ohAitch) [inline tutorials/hoon/landscape-tile.md]: Is using prefix `~` for non-shipnames a landscape thing? Seems likely to get confusing if someone brings up a `simple-walrus` app(letter swap away from valid @p)

Comment (matildepark) [reply tutorials/hoon/landscape-tile.md]: all landscape applications have `~` prefixes in their endpoint

**Comment by matildepark** [comment]
It would be cool if these talked a bit about using `api.bind` — peers are not uncommon in userspace apps and it's something I'm working through at the moment, myself.

**Comment by ohAitch** [comment]
An overview of how you'd explicitly achieve the same component state binding that the tile system provides would go a long way towards explaining both!

How out of date are the api descriptions in the [original eyre doc](https://urbit.org/docs/tutorials/arvo/eyre/#post-authentication-app-communication) / at the bottom of the [gall one](https://urbit.org/docs/tutorials/arvo/gall/)?

File: tutorials/hoon/landscape-tile.md [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+    constructor(props) {
+        super(props);
+        console.log("og props");
+        console.log(this.props);
+        this.state = { ship: "~zod" };
+    }
+
+    sub() {
+        api.action('testing', 'json', {ship: this.state.ship});
+    }
```
[the remark this answers is not in the document]
Comment (ohAitch) [reply tutorials/hoon/landscape-tile.md]: Like `sendHi` for example.

**Comment by baudtack** [comment]
> I didn't actually mention `this` but agree it can't hurt!

Frankly, I hate it. but that's an argument for another time. It at least needs to be an alias.

File: tutorials/hoon/landscape-tile.md [hunk 6]
```diff
@@ -0,0 +1,278 @@
...
```
Comment (ohAitch) [inline tutorials/hoon/landscape-tile.md]: Like `sendHi` for example.

**Pull request merged**